authentication/auth.go

```diff
@@ -251,7 +261,28 @@ func launcherSchemaFromURL(url string) (launcherSchema surveys.LauncherSchema) {
 		URL:      url + cacheBust,
 	}
 
-	return launcherSchema
+	return launcherSchema, ""
+}
+
+func validateSchema(payload []byte) (error string) {
+	if settings.Get("SCHEMA_VALIDATOR_URL") != "" {
+		validateURL := settings.Get("SCHEMA_VALIDATOR_URL") + "/validate"
```

Comment:
is there some `path.join()` type function, so that it will work if there is (or isn't) a trailing slash on `SCHEMA_VALIDATOR_URL`?

Reply:
https://golang.org/pkg/path/#Join

Reply:
👍